setup.py

```diff
@@ -28,7 +28,7 @@
     long_description=long_description,
     long_description_content_type='text/markdown',
     url='https://github.com/RWTH-EBC/AixCaliBuHA',
-    download_url='https://github.com/RWTH-EBC/AixCaliBuHA/archive/refs/tags/0.2.2.tar.gz',
+    download_url='https://github.com/RWTH-EBC/AixCaliBuHA/archive/refs/tags/0.2.3.tar.gz',
```

Comment:
can you use `f'https://github.com/RWTH-EBC/AixCaliBuHA/archive/refs/tags/{__version__}.tar.gz'` here?

Reply:
Nice! Yes!!